web-client/iron-remote-gui/public/test.html

```diff
@@ -1,19 +1,19 @@
-﻿<!DOCTYPE html>
+﻿<!doctype html>
```

Comment:
This was actually the default for prettier, and some big projects such as bootstrap were already using lowercase since before the default changed: https://github.com/twbs/bootstrap/pull/24217
I don’t feel like fighting against that, let’s just stick to what is considered to be a good default by prettier